modules/navigation/3d/nav_mesh_queries_3d.h

```diff
@@ -78,6 +78,11 @@ class NavMeshQueries3D {
 		Vector3 map_up;
 		NavMap *map = nullptr;
 		PathQuerySlot *path_query_slot = nullptr;
+		uint32_t link_polygons_size = 0;
+		const gd::Polygon *begin_poly = nullptr;
+		const gd::Polygon *end_poly = nullptr;
+		Vector3 begin_point;
```

Comment:
Maybe add some comments here, or an overview, of what those are used for. It's not clear what is the difference between `begin_point` and `begin_position`

Reply:
Need to change that. This is actual leftover from trying to rename those in the original PR.